test.sh

```diff
@@ -61,7 +61,7 @@ function run_test_coverage() {
   # We don't use the run function here because sometimes goveralls fails to
   # contact the server and exits with non-zero status, but we don't want to
   # treat that as a failure.
-  goveralls -v -coverprofile=gover.coverprofile -service=travis-ci
```

Comment:
Is this `travis-ci` -> `travis-pro` change to cover the migration to travis.com? I don't actually know what travis-pro is. Could you document the answer in the PR description? Also, reminder to update the description to reflect the latest changes.

Reply:
Ah, yes, this is meant to correspond to travis-ci.com, I forgot to update the description with this. On a second look though I'm not sure it actually does anything relevant. It's documented in https://docs.coveralls.io/supported-ci-services:

>Another important option is `service_name` which allows you to specify where Coveralls should look to find additional information about your builds. This can be any string, but using `travis-ci` or `travis-pro` will allow Coveralls to fetch branch data, comment on pull requests, and more.

However, on second reading, it seems like using *either* of them will trigger the Travis integration, and I'm guessing the Travis API is unified and will happily return data no matter which API endpoint you use. So this might not actually have any practical effect, but it's probably still nice to set it.

Reply:
Updated PR description.